### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by rabah-khalek**

### CODE REVIEW

File: giskard_vision/landmark_detection/types.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import numpy as np
+
+from giskard_vision.core.types import (
+    IMAGE_TYPE,
+    META_TYPE,
+    PredictionResultBase,
+    TypesBase,
+)
+
+LANDMARK_LABEL_TYPE = np.ndarray
```
Comment (Inokinoki) [inline giskard_vision/landmark_detection/types.py:13]: Our previous use case has only one face per image, should we support multiple faces as well?

I think here `np.ndarray` is an array of landmarks with fixed sizes. Maybe let's add a comment for it in any cases (multiple faces/single face)

Comment (Inokinoki) [reply giskard_vision/landmark_detection/types.py:13]: As well as the description of the label, which are actually a list of landmarks.

**Review by Inokinoki — APPROVED** [review]
LGTM

What is the scope of object_detection? If we are trying to define the boundary boxes and the labels of the objects in the image, maybe we can reduce the landmark detection as a special case of object detection as well: boundary boxes become the points of the landmarks, and the orders/locations of the landmarks can be the labels.
But this is not a must anyway.

Further in the object detection, there could be some networks outputting segment masks. Is the image segmentation also in scope?

File: .github/workflows/build-python.yml [hunk 2]
```diff
@@ -66,6 +66,7 @@ jobs:
           pip uninstall opencv-contrib-python
           pip install opencv-contrib-python
 
+
```
Comment (rabah-khalek) [inline .github/workflows/build-python.yml:69]:
```suggestion
```

File: giskard_vision/landmark_detection/types.py (hunk 1 above)
Comment (rabah-khalek) [inline giskard_vision/landmark_detection/types.py:13]:
```suggestion
LANDMARK_LABEL_TYPE = np.ndarray # Just one face is supported
```

**Pull request merged**